### DISCUSSION TITLE
Philips Hue White and Colour Ambiance G95 – E27 smart bulb

### DISCUSSION THREAD

**Discussion by matt220992** [Koenkk/zigbee2mqtt#24301] [Devices]
Very much ready to get shot down for not reading various things on how all this works - but love this integration with Home Assistant etc but am very much a novice - I find GitHub generally a bit over my head!

Just wanted to make aware that the new Philips Hue G95 lamp doesn't seem to have support in zigbee2mqtt yet. It is quite a new model, so just wanted to make the right people aware so it might get added.
It seems to function but doesn't pop up as other products do, just has a ZigBee logo and unsupported written next to it.

If I can provide information or help to those that can fix this I will gladly do my best to help if told exactly what to do! haha

**Comment by Koenkk** [comment]
see https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html how to add support for new devices
